recreating issue with fontstruct otfs and (u?)harfbuzz after 0.39.1

`uv run main.py`

(or other equivalent with a python 3.12 and uharfbuzz 0.39.3/0.39.1)